src/main/java/com/cascadebot/cascadebot/commands/core/UserInfoCommand.java

```diff
@@ -38,11 +39,13 @@ public void onCommand(Member sender, CommandContext context) {
         builder.setTitle(DiscordUtils.getTag(user));
         builder.setThumbnail(user.getAvatarUrl());
         builder.addField("Crated Data", "TODO", true);
-        builder.addField("Join data", "TODO", true);
-
+        builder.addField("Join Data", "TODO", true);
```

Comment:
When I said add this back, you need to actually fill it out 😂

Reply:
all done :)